pkg/apis/cnsoperator/storagepolicy/v1alpha2/doc.go

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
We should not be using this.  It should be consistent with copyrights in other files.
https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/csi/service/vanilla/controller.go#L2
Note that vsphere-csi-driver is a project under kubernetes-sigs org.

cc @divyenpatel

Reply:
done